components/src/core/AutoComplete/AutoComplete.tsx

```diff
@@ -0,0 +1,356 @@
+                setTextInput('');
+                if (onChange) {
+                    onChange(newChip);
+                }
+            }
+        }
+    };
+    const onTagsSelected = (tag: string): void => {
+        if (chipValue.length < limitTags) {
+            const newChip = chipValue;
+            newChip.push(tag);
+            setChipValue(newChip);
+            setFilterOptions(filterChips(options, newChip));
+            setTextInput('');
+            if (onChange) {
+                onChange(newChip);
+            }
+        }
+    };
+    const removeChipItem = (item: string): void => {
+        const arr = chipValue.filter((str) => str !== item);
+        setChipValue(arr);
+        setFilterOptions(filterChips(options, arr));
+        if (onDelete) {
+            onDelete(item);
+        }
+    };
+
+    return (
+        <View style={[defaultStyles.individualTextInputWrapper, defaultStyles.tagInputWrapper, styles?.root]}>
+            <View>
+                <TouchableHighlight onPress={handleTextInputPress}>
+                    <View style={defaultStyles.selectorContainer}>
+                        <View style={{ width: '93%' }}>
+                            {!hideDropDownTags || chipValue.length > 0 ? (
+                                <HelperText style={defaultStyles.labelStyle} type="info">
+                                    {!hideDropDownTags || chipValue.length > 0 ? label : ''}{' '}
+                                </HelperText>
+                            ) : null}
+                            <View style={[defaultStyles.tagInput, styles?.textInputContainer]}>
+                                {chipValue.map((item) => (
+                                    <Chip
+                                        key={item}
+                                        style={[defaultStyles.chip, styles?.chip]}
+                                        borderColor={theme.colors.outline}
+                                        textColor={theme.colors.onSurfaceVariant}
+                                        disabled={disabled}
+                                        onClose={(): void => {
+                                            removeChipItem(item);
+                                        }}
+                                        {...chipProps}
+                                    >
+                                        {item}
+                                    </Chip>
+                                ))}
+
+                                <RNTextInput
+                                    testID="tagInput"
+                                    ref={tagInputRef}
+                                    selectionColor={theme.colors.primary}
+                                    value={textInput}
+                                    placeholderTextColor={
+                                        hideDropDownTags ? theme.colors.onSurfaceVariant : 'rgba(255, 255, 255, 0)'
```

Comment:
rgba(255, 255, 255, 0) will this look good when theme is dark or do we need to find the respective `theme.colors` ?